src/Compilers/VisualBasic/Test/Symbol/CrossLanguageTest.vb

```diff
@@ -0,0 +1,50 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class CrossLanguageTest
+        Inherits BasicTestBase
+
+        <Fact>
+        Public Sub CanAchieveHandledEvents()
+            Dim csharpTree = CSharp.CSharpSyntaxTree.ParseText(
+                "public class Sample
+                {
+                    public void Create()
+                    {
+                    }
+                }")
+
+            Dim csharpCompilation = CSharp.CSharpCompilation.Create(
+                "Test",
+                {csharpTree},
+                {MscorlibRef_v4_0_30316_17626},
+                New CSharp.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary))
+
+            Dim methods = csharpCompilation.GetTypeByMetadataName("Sample").GetMembers("Create").OfType(Of IMethodSymbol).ToList()
+
+            Assert.True(methods.Count = 1)
```

Comment:
>methods.Count = 1 [](start = 24, length = 17)

Instead of inlining equality comparison, please use Assert.Equal helper. For example, in this case - ```Assert.Equal(1, methods.Count)```